### ISSUE TITLE
xfce4-sensors-plugin-1.3.92 doesn't compile with LTO

### ISSUE THREAD

**Issue by ElDavoo** [InBetweenNames/gentooLTO#249]
You need to disable lto completely to make it compile, otherwise ld will get a segmentation fault. Everything else (graphite, ipa, o3) works correctly.

[build.log](https://github.com/InBetweenNames/gentooLTO/files/2883852/build.log)
[info.txt](https://github.com/InBetweenNames/gentooLTO/files/2883854/info.txt)
[pretend.txt](https://github.com/InBetweenNames/gentooLTO/files/2883855/pretend.txt)

**Comment by InBetweenNames** [comment]
I actually installed XFCE just to test this -- using GCC 8.3.0 and binutils 2.32 there are no issues.  Can you try with this configuration?

**Comment by ElDavoo** [comment]
Yeah, I can confirm it now works. Strange as I could reproduce it 100% of th etime.

**Comment by ElDavoo** [comment]
Hmmm, strangely enough, after an emptytree with fipa-pta and gcc9 it now fails again, with the exact same error.

**Comment by javashin** [comment]
True

**Comment by javashin** [comment]
i have same problem i was going to open a new issue glad i found it segsfault here outside portage and with portage very strange any fix ?
